feature/bookmarks/src/main/java/com/google/samples/apps/nowinandroid/feature/bookmarks/BookmarksScreen.kt

```diff
@@ -113,7 +111,6 @@ internal fun BookmarksScreen(
 ) {
     val bookmarkRemovedMessage = stringResource(id = R.string.bookmark_removed)
     val undoText = stringResource(id = R.string.undo)
-    val snackbarHostState = remember { SnackbarHostState() }
```

Comment:
Curious to know the reason to extract the snackbarHostState to its parameter.

Was it a requirement of the list/detail implementation or is it part of another clean up?

Reply:
It was part of another, but related clean up. Previously, the BookmarksScreen used a nested Scaffold. I removed that nested scaffold, hence the snackbar statemust come from the root scaffold itself now.

Reply:
Sounds good, thanks!